internal/notification/client.go

```diff
@@ -0,0 +1,110 @@
+package notification
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/opendevstack/pipeline/pkg/logging"
+	"github.com/opendevstack/pipeline/pkg/pipelinectxt"
+)
+
+const (
+	ConfigMapName           = "ods-notification"
+	UrlProperty             = "URL"
+	MethodProperty          = "Method"
+	ContentTypeProperty     = "ContentType"
+	RequestTemplateProperty = "requestTemplate"
+	NotifyOnStatusProperty  = "NotifyOnStatus"
+	EnabledProperty         = "Enabled"
```

Comment:
Shouldn't the properties all be lowercase? The `ConfigMap` keys are all lowercase.

Also, it would be nice to move these to the `config`.go` file as they belong to that.